CRM/Lineitemedit/Form/Edit.php

```diff
@@ -239,7 +249,7 @@ protected function updateEntityRecord($lineItem) {
       $line = array();
       $lineTotal = 0;
       foreach ($getUpdatedLineItems as $updatedLineItem) {
-        $line[$updatedLineItem['price_field_value_id']] = $updatedLineItem['label'] . ' - ' . (float) $updatedLineItem['qty'];
```

Comment:
@monishdeb please make sure to test this change

Reply:
Just to be clear: the `$line` array, created above on L239, appears to only be used once, during an implode operation on L246.
There is side effect here in that the contents of the (text) fee_level field will no longer be sorted by the price field value id, but not sure if that arrangement was better than "the order in which participant line items were added to the database"